model_processing/crates/webify_models/src/cli/create_progress_bar.rs

```diff
@@ -0,0 +1,33 @@
+//! Progress bar factory function, for all your progress bar needs
+
+use indicatif::{ProgressBar, ProgressStyle};
+
+pub fn create_progress_bar(item_count: u64) -> ProgressBar {
+    let bar = ProgressBar::new(item_count);
+    let bar_style: ProgressStyle = ProgressStyle::default_bar()
```

Comment:
It runs fine without it. There's no real reason to put the type annotation there else than just extra clarity? There's no real rhyme or reason why it would have it or not though, I agree.